### ISSUE TITLE
Pre-work Checklist: Developer: Leon Dawson

### ISSUE THREAD

**Issue by ldaws003** [hackforla/website#3611]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ldaws003** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It took about 3.5 hours.

**Comment by ExperimentsInHonesty** [comment]
Hi @ldaws003  if you have not given up on us yet and are still waiting on a [medium front end issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-7198257). There are 4 of them now available

**Comment by ldaws003** [comment]
Hi Bonnie. I won't be able to continue volunteering with Hack For LA for the time being due to school and getting a job.

**Comment by ExperimentsInHonesty** [comment]
Ok, thanks for letting us know.  If you want to pick back up in the future, go ahead and reopen this issue and more to in progress.

**Comment by HackforLABot** [comment]
## Activity Log: ldaws003
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- ldaws003 issue closed by PR 3432: https://github.com/hackforla/website/issues/2113#event-7095774800 at 07/30/2022, 07:47 PM PDT
- ldaws003 opened: https://github.com/hackforla/website/issues/3611 at 10/10/2022, 07:54 PM PDT
- ldaws003 assigned: https://github.com/hackforla/website/issues/2853#issuecomment-1048338245 at 10/24/2022, 03:22 PM PDT
- ldaws003 commented on: https://github.com/hackforla/website/issues/2853#issuecomment-1289710056 at 10/24/2022, 03:26 PM PDT
- ldaws003 opened: https://github.com/hackforla/website/pull/3658 at 10/25/2022, 12:40 AM PDT
- ldaws003 commented on: https://github.com/hackforla/website/issues/3611#issuecomment-1290131013 at 10/25/2022, 12:45 AM PDT
- ldaws003 PR was merged: https://github.com/hackforla/website/pull/3658#event-7668013587 at 10/25/2022, 05:10 PM PDT
- ldaws003 issue closed by PR 3658: https://github.com/hackforla/website/issues/2853#event-7668013633 at 10/25/2022, 05:10 PM PDT
- ldaws003 commented on: https://github.com/hackforla/website/issues/3210#issuecomment-1295956662 at 10/29/2022, 01:01 PM PDT
- ldaws003 assigned: https://github.com/hackforla/website/issues/3210#issuecomment-1295956662 at 10/29/2022, 01:01 PM PDT
- ldaws003 closed as completed: https://github.com/hackforla/website/issues/3611#event-7698517039 at 10/30/2022, 01:14 AM PDT
- ldaws003 opened: https://github.com/hackforla/website/pull/3676 at 10/30/2022, 01:50 AM PDT
- ldaws003 closed: https://github.com/hackforla/website/pull/3676#event-7703173831 at 10/31/2022, 03:58 AM PDT
- ldaws003 opened: https://github.com/hackforla/website/pull/3679 at 10/31/2022, 12:51 PM PDT
- ldaws003 unassigned: https://github.com/hackforla/website/issues/2853#issuecomment-1297644812 at 10/31/2022, 01:34 PM PDT
- ldaws003 PR was merged: https://github.com/hackforla/website/pull/3679#event-7709993235 at 10/31/2022, 09:21 PM PDT
- ldaws003 issue closed by PR 3679: https://github.com/hackforla/website/issues/3210#event-7709993357 at 10/31/2022, 09:21 PM PDT
- ldaws003 assigned: https://github.com/hackforla/website/issues/2853#issuecomment-1297644812 at 10/31/2022, 10:00 PM PDT
- ldaws003 commented on: https://github.com/hackforla/website/issues/2853#issuecomment-1298030446 at 10/31/2022, 10:01 PM PDT
- ldaws003 opened: https://github.com/hackforla/website/pull/3686 at 10/31/2022, 10:25 PM PDT
- ldaws003 PR was merged: https://github.com/hackforla/website/pull/3686#event-7719210329 at 11/01/2022, 09:14 PM PDT
- ldaws003 issue closed by PR 3658: https://github.com/hackforla/website/issues/2853#event-7719210458 at 11/01/2022, 09:14 PM PDT
- ldaws003 assigned: https://github.com/hackforla/website/issues/2113#issuecomment-1304867526 at 11/08/2022, 04:13 AM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2113#issuecomment-1307116523 at 11/08/2022, 04:14 AM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2113#issuecomment-1308266365 at 11/08/2022, 10:16 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2113#issuecomment-1311386173 at 11/11/2022, 12:29 AM PST
- ldaws003 opened: https://github.com/hackforla/website/pull/3716 at 11/11/2022, 11:56 PM PST
- ldaws003 PR was merged: https://github.com/hackforla/website/pull/3716#event-7797609429 at 11/12/2022, 06:02 PM PST
- ldaws003 issue closed by PR 3432: https://github.com/hackforla/website/issues/2113#event-7797609454 at 11/12/2022, 06:02 PM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/2675#issuecomment-1192160542 at 11/12/2022, 08:13 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2675#issuecomment-1312635848 at 11/12/2022, 08:20 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2675#issuecomment-1321062211 at 11/19/2022, 11:44 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2675#issuecomment-1328312763 at 11/27/2022, 10:32 AM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2675#issuecomment-1336518335 at 12/04/2022, 01:11 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2675#issuecomment-1345762979 at 12/11/2022, 06:04 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/2675#issuecomment-1363823785 at 12/23/2022, 02:25 AM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/3223#issuecomment-1236197698 at 01/25/2023, 02:43 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1404328769 at 01/25/2023, 02:52 PM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1416739561 at 02/04/2023, 04:28 AM PST
- ldaws003 commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1435875548 at 02/18/2023, 09:38 PM PST
- ldaws003 issue closed as completed: https://github.com/hackforla/website/issues/3611#event-8622182252 at 02/27/2023, 06:35 PM PST
- ldaws003 unassigned: https://github.com/hackforla/website/issues/3223#issuecomment-1453087219 at 03/08/2023, 02:55 PM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/3611#event-8622182252 at 03/19/2023, 11:15 AM PDT
- ldaws003 commented on: https://github.com/hackforla/website/issues/3611#issuecomment-1479998733 at 03/22/2023, 10:40 AM PDT
- ldaws003 issue closed as completed: https://github.com/hackforla/website/issues/3611#event-8848317207 at 03/26/2023, 03:58 PM PDT
- ldaws003 opened: https://github.com/hackforla/website/issues/8403 at 11/04/2025, 08:22 PM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/8403 at 11/04/2025, 08:22 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3489282370 at 11/04/2025, 08:38 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3507427373 at 11/08/2025, 06:59 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3507759231 at 11/09/2025, 01:00 AM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/7418 at 11/09/2025, 02:23 AM PST
- ldaws003 unassigned: https://github.com/hackforla/website/issues/7418 at 11/09/2025, 02:23 AM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/7333 at 11/10/2025, 05:30 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/7333#issuecomment-3511692242 at 11/10/2025, 05:33 AM PST
- ldaws003 opened: https://github.com/hackforla/website/pull/8416 at 11/10/2025, 11:11 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3515320637 at 11/10/2025, 11:24 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3519748228 at 11/11/2025, 07:36 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3519754378 at 11/11/2025, 07:39 PM PST
- ldaws003 PR was merged: https://github.com/hackforla/website/pull/8416 at 11/12/2025, 07:20 PM PST
- ldaws003 issue was closed as completed: https://github.com/hackforla/website/issues/7333 at 11/12/2025, 07:20 PM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/7337 at 11/12/2025, 11:14 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/7337#issuecomment-3525965168 at 11/12/2025, 11:15 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3525972112 at 11/12/2025, 11:17 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3525973561 at 11/12/2025, 11:17 PM PST
- ldaws003 opened: https://github.com/hackforla/website/pull/8418 at 11/13/2025, 01:23 AM PST
- ldaws003 commented: https://github.com/hackforla/website/pull/8418#issuecomment-3529908111 at 11/13/2025, 02:07 PM PST
- ldaws003 PR was merged: https://github.com/hackforla/website/pull/8418 at 11/15/2025, 08:02 PM PST
- ldaws003 issue was closed as completed: https://github.com/hackforla/website/issues/7337 at 11/15/2025, 08:02 PM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/7783 at 11/17/2025, 11:56 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/7783#issuecomment-3543615499 at 11/17/2025, 11:58 AM PST
- ldaws003 issue was closed as completed: https://github.com/hackforla/website/issues/7783 at 11/22/2025, 10:03 AM PST
- ldaws003 commented: https://github.com/hackforla/website/pull/8435#issuecomment-3567596544 at 11/22/2025, 11:59 PM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/3328 at 11/23/2025, 12:04 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/3328#issuecomment-3567602196 at 11/23/2025, 12:07 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3569917056 at 11/24/2025, 02:11 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3569937661 at 11/24/2025, 02:15 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/3328#issuecomment-3571617471 at 11/24/2025, 08:19 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/3328#issuecomment-3576297593 at 11/25/2025, 07:41 AM PST
- ldaws003 opened: https://github.com/hackforla/website/pull/8437 at 11/25/2025, 08:57 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3576673193 at 11/25/2025, 09:10 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3576696789 at 11/25/2025, 09:16 AM PST
- ldaws003 commented: https://github.com/hackforla/website/pull/8437#issuecomment-3621369780 at 12/06/2025, 03:38 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/3328#issuecomment-3621380694 at 12/06/2025, 03:53 PM PST
- ldaws003 commented: https://github.com/hackforla/website/pull/8446#issuecomment-3647957318 at 12/12/2025, 12:00 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3648484235 at 12/12/2025, 03:20 PM PST
- ldaws003 commented: https://github.com/hackforla/website/pull/8446#issuecomment-3648621608 at 12/12/2025, 04:45 PM PST
- ldaws003 commented: https://github.com/hackforla/website/pull/8446#issuecomment-3663538490 at 12/16/2025, 08:02 PM PST
- ldaws003 issue was closed as completed: https://github.com/hackforla/website/issues/3328 at 01/10/2026, 01:44 PM PST
- ldaws003 PR was merged: https://github.com/hackforla/website/pull/8437 at 01/10/2026, 01:44 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3761475448 at 01/16/2026, 11:26 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/3328#issuecomment-3771636178 at 01/20/2026, 12:27 AM PST
- ldaws003 assigned: https://github.com/hackforla/website/issues/8288 at 01/20/2026, 12:30 AM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8288#issuecomment-3771667223 at 01/20/2026, 12:36 AM PST
- ldaws003 opened: https://github.com/hackforla/website/issues/8470 at 01/22/2026, 11:08 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/3328#issuecomment-3788679125 at 01/22/2026, 11:09 PM PST
- ldaws003 commented: https://github.com/hackforla/website/pull/8471#issuecomment-3793155523 at 01/23/2026, 04:29 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3795661254 at 01/24/2026, 02:43 PM PST
- ldaws003 opened: https://github.com/hackforla/website/issues/8476 at 01/27/2026, 03:08 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8288#issuecomment-3808034332 at 01/27/2026, 03:13 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8288#issuecomment-3808039829 at 01/27/2026, 03:14 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8288#issuecomment-3808427318 at 01/27/2026, 05:20 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-3954880359 at 02/24/2026, 01:38 PM PST
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4189587655 at 04/05/2026, 02:52 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4272371859 at 04/17/2026, 06:59 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8288#issuecomment-4364840847 at 05/02/2026, 03:18 PM PDT
- ldaws003 opened: https://github.com/hackforla/website/issues/8628 at 05/02/2026, 04:36 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4364979284 at 05/02/2026, 04:53 PM PDT
- ldaws003 issue was closed as completed: https://github.com/hackforla/website/issues/8288 at 05/12/2026, 07:44 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4526796184 at 05/23/2026, 04:24 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/pull/8661#issuecomment-4581536276 at 05/29/2026, 08:41 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4586653616 at 05/31/2026, 05:13 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4761356969 at 06/21/2026, 01:15 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/pull/8699#issuecomment-4761363184 at 06/21/2026, 01:18 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4763410330 at 06/21/2026, 02:57 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/pull/8710#issuecomment-4945042586 at 07/11/2026, 03:43 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-4950488800 at 07/12/2026, 01:13 AM PDT
- ldaws003 assigned: https://github.com/hackforla/website/issues/8466 at 08/19/2026, 06:30 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-5385022351 at 08/23/2026, 01:18 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8466#issuecomment-5385027237 at 08/23/2026, 01:19 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/pull/8775#issuecomment-5561792778 at 09/06/2026, 12:59 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-5570357333 at 09/07/2026, 05:05 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8466#issuecomment-5570403766 at 09/07/2026, 05:09 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-5625145009 at 09/10/2026, 01:40 PM PDT
- ldaws003 opened: https://github.com/hackforla/website/pull/8781 at 09/13/2026, 05:22 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8466#issuecomment-5718375296 at 09/17/2026, 10:12 AM PDT
- ldaws003 commented: https://github.com/hackforla/website/issues/8403#issuecomment-5737919544 at 09/18/2026, 05:38 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/pull/8781#issuecomment-5746733051 at 09/19/2026, 06:31 PM PDT
- ldaws003 commented: https://github.com/hackforla/website/pull/8797#issuecomment-5783525362 at 09/22/2026, 01:21 PM PDT
